tests/v2/actions/workloads/deployment/deployment.go

```diff
@@ -88,3 +97,41 @@ func UpdateDeployment(client *rancher.Client, clusterID, namespaceName string, d
 
 	return updatedDeployment, err
 }
+
+// RolbackDeployment is a helper to rollback deployments
+func RollbackDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment, revision int) (string, error) {
```

Comment:
all you need is the deployment name, right? not the whole deployment?

Reply:
Yes, I can change the parameters to get the deployment Name, but I also like the idea of pass the deployment and function is responsible to get the needed deployment informations

Reply:
okay I'll leave it up to you then. But usually we want to keep the params within scope, so if it isn't using the full deployment it would make sense to just send what the func needs.

Reply:
I'll add the parameter